### ISSUE TITLE
Build slides for making your first contribution

### ISSUE THREAD

**Issue by jimbobbennett** [open-source-bootcamp/bootcamp-guide#5]
We need a slide deck to walk event attendees through making their first contribution to an open source project using CubeFlyer.

This would need to cover:

* Forking
  * Explaining forks
  * Dividing the attendees into small teams (3/4 people), and nominating one person to 'own' the golden source repo. Teams are important as we are teaching how to collaborate, but if folks don't want to be in a team they can listen to learn about merge conflicts.
  * Having the owner create a new repo from https://github.com/open-source-bootcamp/CubeFlyer
  * Getting the rest of the team to fork this repo
  * A brief walkthrough of the code
  * A demo showing how to run the code
* Hands on time
  * Getting attendees to play the game
* Issues
  * Explaining issues in GitHub
  * Getting the owner to create a new issue asking for new greetings
  * Showing the issue exists only in the original repo, not the fork
* Hands on time
  * Getting attendees to add one or more greetings
  * Testing the greetings
* Pull requests and merging as a team
  * Getting a first pull request in and merged
  * Getting a second pull request in and seeing it blocked by a merge conflict
* Merge conflicts
  * Explaining merge conflicts with examples
  * Showing how to fix merge conflicts in VS Code by editing the file
  * Pushing the fixed version into the Pull Request
  * Getting everyone to get their PRs in

**Comment by runewake2** [comment]
I typically build my slides in [Reveal JS](https://revealjs.com/) using Markdown. Is this OK or is there a standard library you want us to use?

**Comment by runewake2** [comment]
This will require the presenter to be familiar with and have Node installed it seems. Is that OK?

If so, maybe we should host these slides in a static site so alt-text and notes are accessible.

**Comment by jimbobbennett** [comment]
I don't think that's too much to ask for a presenter to install Node if they don't already have it.

We could host them locally, but that would remove the ability to customize with presenter details.

**Comment by jimbobbennett** [comment]
FYI - fixed the mermaid issues. Needed some version bumps on packages. There is a weird issue now with the latest version where in presenter view the [SliDev logo is enormous](https://github.com/slidevjs/slidev/issues/733), but this is a known issue and hopefully will be fixed soon!

**Comment by jimbobbennett** [comment]
Closed by #24
